README.md

```diff
@@ -1 +1,460 @@
-# golangcodestyle
\ No newline at end of file
+# Backend (Golang) Coding Guidelines
```

Comment:
4. Use counterfeiter (https://github.com/maxbrunsfeld/counterfeiter) to generate the mocks.

Reply:
added: https://github.com/apaliavy/golangcodestyle/blob/1eda2a9506765ca11274f4bf973baf16a84174af/README.md#mocks